tools/parabricks/parabricks_fq2bam.xml

```diff
@@ -0,0 +1,123 @@
+<tool id="parabricks_fq2bam" name="Parabricks fq2bam" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>Run GPU-bwa mem mapper</description>
+    <macros>
+        <token name="@TOOL_VERSION@">4.4.0-1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+        <token name="@PROFILE@">24.2</token>
+    </macros>
+    <creator>
+        <organization name="European Galaxy Team" url="https://galaxyproject.org/eu/"/>
+        <person givenName="Anup" familyName="Kumar" email="kumara@informatik.uni-freiburg.de"/>
+    </creator>
+    <requirements>
+        <container type="docker">nvcr.io/nvidia/clara/clara-parabricks:@TOOL_VERSION@</container>
+    </requirements>
+    <command detect_errors="aggressive">
+    <![CDATA[
+        #if $reference_source.reference_source_selector == 'cached':
+            ln -f -s '$reference_source.ref_file.fields.path' reference.fasta && 
+        #end if
+        #set $o_mappings = 'output_mapped_file.bam'
+        ln -f -s '$fastq_input.fastq_input1' input1.fastq &&
+        #if $fastq_input.fastq_input_selector == 'paired':
+            ln -f -s '$fastq_input.fastq_input2' input2.fastq &&
+            pbrun fq2bam --low-memory --ref reference.fasta --in-fq input1.fastq input2.fastq --out-bam '${o_mappings}' &&
+        #else:
+            pbrun fq2bam --low-memory --ref reference.fasta --in-se-fq input1.fastq --out-bam '${o_mappings}' &&
+        #end if  
+        cp '${o_mappings}' '$output_mapping'
+    ]]>
+    </command>
+    <inputs>
+        <conditional name="reference_source">
+            <param name="reference_source_selector" type="select" label="Will you select a reference genome from your history or use a built-in index?" help="Built-ins were indexed using default options. If you would like to perform self-mapping select `history` here, then choose your input file as reference (currently not supported).">
+                <option value="cached">Use a built-in genome index</option>
+            </param>
+            <when value="cached">
+                <param name="ref_file" type="select" label="Using reference genome" help="Select genome from the list">
+                    <options from_data_table="all_fasta">
```

Comment:
as above, the discussion was to make all pre-built indices available to the tool so you need to choose the correct data table here.